### PR TITLE
Updates teaser text in hub page to act as link

### PR DESCRIPTION
**Pull request by RobAWilkinson**
## Description
This is a WIP branch that updates the teaser text on hub pages to make the text clickable.
Would like to get input from @emilywaggoner and @andrewlewandowski on the transition time of the external link icon, adjust finalize and then merge from there.

## Testing done
Local testing

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: va-gov/components/merger-crosslinks.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h3 class="va-nav-linkslist-heading">{{ group.heading }}</h3>
           <li>
             <a href="{{link.url}}">
             {% if link.title != empty %}
-              {{ link.title }}
+              {{ link.title }}<img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
```
Comment (jbalboni) [inline va-gov/components/merger-crosslinks.html]: We probably want to add `aria-hidden` to these arrows. They're not useful content for screen readers.

**Comment by RobAWilkinson** [comment]
I took the approach to creating a block and documenting it in the design system here: https://github.com/department-of-veterans-affairs/design-system/pull/190/files
then standardized everything on all the hub pages to use that markup

**Comment by ncksllvn** [comment]
Oh no, I just realized that this pull request contains changes that will need to go into the `vagov-content` repo! Since this is a recent PR, we can probably just copy over the contents into that repo and not have any problems

**Comment by RobAWilkinson** [comment]
@ncksllvn would love your input on this, brought all these links on the hub page into a new metalsmith component. Accordingly a ton of changes in the markdown files, should I open PR on vagov-content?

File: va-gov/components/hub-page-link-list.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          {% endif %}
+        </span>
+        <p class="hub-page-link-list__description">
+        {{ link.description }}
+        </p>
+      </a>
+    </li>
+    {% endfor %}
+  </ul>
+{% endfor %}
```
Comment (ncksllvn) [inline va-gov/components/hub-page-link-list.html]: 🔥🔥🔥

File: va-gov/includes/level2-index.html [hunk 3]
```diff
@@ -20,6 +20,11 @@ <h1>
       </div>
       {{ contents }}
 
+      {% if hublinks != empty %}
+        {% include "va-gov/components/hub-page-link-list.html" %}
+      {% endif %}
```
Comment (ncksllvn) [inline va-gov/includes/level2-index.html]: `empty` - So we’re all becoming experts at Liquid templates at this point 😆

File: va-gov/components/hub-page-link-list.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- url:
+  label:
+  description:
+  external: true/false
+
+{% endcomment %}
+{% for hublink in hublinks %}
+  <section class='usa-grid'>
+    <div class="va-h-ruled--stars"></div>
+  </section>
```
Comment (ncksllvn) [inline va-gov/components/hub-page-link-list.html]: Is `section` the right tag to be used here?

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Comment by ncksllvn** [comment]
Huge improvement @RobAWilkinson! Thank you for doing this. Will be hugely appreciated especially once changes are starting to be made to the hubs.

File: va-gov/components/hub-page-link-list.html (hunk 4 above)
Comment (RobAWilkinson) [reply va-gov/components/hub-page-link-list.html]: honestly maybe the best would be an `hr` tag with a class which is exactly what it is

**Pull request merged**